service/src/main/java/bio/terra/cbas/dependencies/dockstore/DockstoreService.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.cbas.dependencies.dockstore;
+
+import bio.terra.dockstore.api.Ga4Ghv1Api;
+import bio.terra.dockstore.client.ApiException;
+import bio.terra.dockstore.model.ToolDescriptor;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DockstoreService {
+  private final DockstoreClient dockstoreClient;
+  private static final String WDL_TYPE = "WDL";
+  private static final String WORKFLOW_IDENTIFIER = "#workflow/";
+
+  public DockstoreService(DockstoreClient dockstoreClient) {
+    this.dockstoreClient = dockstoreClient;
+  }
+
+  private Ga4Ghv1Api ga4Ghv1Api() {
+    return new Ga4Ghv1Api(dockstoreClient.getApiClient());
```

Comment:
In the other dependencies I think we put these API getters into the equivalent of `DockstoreClient.java`. Not sure whether that makes much difference... but I guess it might make dependency injection harder in certain test cases?

Reply:
I was following what we did for [Leonardo AppsV2Api](https://github.com/DataBiosphere/cbas/blob/dbcc106e4edfbd9daad116a8d64a8f6afd7d9d3a/service/src/main/java/bio/terra/cbas/dependencies/leonardo/LeonardoService.java#L27). Should I be looking at some other service class?

Reply:
Oh, ok then, never mind! 😆